VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1076,10 +1076,14 @@ export abstract class WaitingRoomBaseDirective {
         logPaylod.reason = 'Not showing video because hearing is not in session and user is not in consultation';
         this.logger.debug(`${this.loggerPrefix} ${logPaylod.reason}`, logPaylod);
         this.showVideo = false;
+        this.conferenceStartedBy = undefined;
         this.showConsultationControls = false;
         this.isPrivateConsultation = false;
     }
 
+    isConferenceStartedBy(): boolean {
```

Comment:
Make more clear that it is checking it was started by the current user.

Reply:
Done